README.md

```diff
@@ -1,2 +1,164 @@
 # github-backup
 :back: :arrow_up: A script to backup your GitHub Account/Organisation so you can still work when (they/you are) offline.
+
+## Why?
+
+As a person who uses GitHub as their "_single-source-of-truth_"
+I need a backup of GitHub issues
+so that I can work "offline".
+(_either when I have no network or GH is "down"_)
+
+## What?
+
+Backup GitHub issues.
+
++ a script that lets you login to GitHub using OAuth
```

Comment:
it's no longer going to be a "script" more of a "_standalone app_" to view GitHub issue revision history.